backend/infrahub/database/__init__.py

```diff
@@ -349,6 +349,8 @@ async def execute_query_with_metadata(
 
             with QUERY_EXECUTION_METRICS.labels(**labels).time():
                 response = await self.run_query(query=query, params=params, name=name)
+                if response is None:
```

Comment:
`run_query` is supposed to return a non-optional `AsyncResult`, when can `response` be `None`?

Reply:
this is a really good question and I've been wondering if this is the right fix for it but without that it was not working and I couldn't figure out why it was returning None